content/negotiation/quest_cards.lua

```diff
@@ -384,6 +384,31 @@ local CARDS = {
             end)
         end,
     },
+    faction_negotiation_hinder =
+    {
+        name = "Faction Hinder",
+        show_dealt = false,
+        quip = "support",
+        rarity = CARD_RARITY.UNIQUE,
+        flags = CARD_FLAGS.BYSTANDER,
+        resolve_scale = {20, 25, 30, 35},
+        OnPostResolve = function( self )
+            local core_id = "POWER_ABUSE"
+            local opposition_data = DemocracyUtil.GetOppositionData(self.owner)
+            if opposition_data and opposition_data.faction_core then
+                core_id = opposition_data.faction_core
+            end
+            local mod = Negotiation.Modifier( core_id, self.negotiator )
+            self.negotiator:CreateModifier( mod )
+            mod:SetResolve(DemocracyUtil.CalculateBossScale(self.resolve_scale))
+
+            self.engine:BroadcastEvent(EVENT.CUSTOM, function(panel)
```

Comment:
We don't need this part for day 4.
For day 3, this is here because there are too many helpers or hinders at once, so the game speeds up and skips a lot of the dialogs. This isn't necessary because there is only one hinder whose dialog isn't necessary.